openam-ui/openam-ui-ria/src/main/js/org/forgerock/openam/ui/admin/views/realms/dataStores/DataStoresView.jsm

```diff
@@ -0,0 +1,114 @@
+/**
+ * The contents of this file are subject to the terms of the Common Development and
+ * Distribution License (the License). You may not use this file except in compliance with the
+ * License.
+ *
+ * You can obtain a copy of the License at legal/CDDLv1.0.txt. See the License for the
+ * specific language governing permission and limitations under the License.
+ *
+ * When distributing Covered Software, include this CDDL Header Notice in each file and include
+ * the License file at legal/CDDLv1.0.txt. If applicable, add the following below the CDDL
+ * Header, with the fields enclosed by brackets [] replaced by your own identifying
+ * information: "Portions copyright [year] [name of copyright owner]".
+ *
+ * Copyright 2024 Wren Security.
+ */
+
+import $ from "jquery";
+import _ from "lodash";
+import Messages from "org/forgerock/commons/ui/common/components/Messages";
+import AbstractView from "org/forgerock/commons/ui/common/main/AbstractView";
+import { getAll, getCreatables, remove } from "org/forgerock/openam/ui/admin/services/realm/DataStoresService";
```

Comment:
I am not a fan of exporting and directly importing methods with such generic names. However, this is already present in other existing ESM components, so this comment is just a rant :)